snippets/en/_includes/self-managed-networking-requirements.mdx

```diff
@@ -0,0 +1,11 @@
+For a networked deployment, egress to these endpoints is required during _both_ installation and runtime:
+* https://deploy.wandb.ai
+* https://charts.wandb.ai
+* https://docker.io
+* https://quay.io
+* https://gcr.io
```

Comment:
Same as the other review, we don't use these two repos in the regular deployment.

Reply:
We actually use the `quay.io` for the Prometheus images